jdbc/pom.xml

```diff
@@ -74,8 +74,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <version>2.7.2</version>
-      <scope>provided</scope>
+      <version>${hadoop.common.version}</version>
     </dependency>
```

Comment:
hadoop.version is x.y.z, see https://github.com/apache/zeppelin/blob/master/spark-dependencies/pom.xml#L602